zh/option/component/geo-common.md

```diff
@@ -89,6 +89,20 @@ boundingCoords: [
 }
 ```
 
+#${prefix} nameProperty(string) = 'name'
+
+默认是 ’name‘，针对 GeoJSON 要素的自定义属性名称，作为主键用于关联数据点和 GeoJSON 地理要素。
+例如： 
+```js
+{
+    nameProperty: 'NAME', // 数据点中的 name：Alabama 会关联到 GeoJSON 中 NAME 属性值为 Alabama 的地理要素{"type":"Feature","id":"01","properties":{"NAME":"Alabama"}, "geometry": { ... }}
+    data:[
+        {name: 'Alabama', value: 4822023},
+        {name: 'Alaska', value: 731449},
+    ]
+}
+```
+
 #${prefix} selectedMode(boolean|string) = false
```

Comment:
The English doc seems to be wrong here with `#${prefix}` being `##`. Do you mind changing in this PR?